### PR TITLE
Super fast reindex_works tasks

### PR DESCRIPTION
**Pull request by jrochkind**
* Only does works to prevent the slower find all fedora ids task
* batches solr adds
* uses a progress bar

### CODE REVIEW

File: lib/tasks/chf_tasks.rake [hunk 1]
```diff
@@ -56,6 +56,31 @@ namespace :chf do
...
+    add_batch_size = ENV['ADD_BATCH_SIZE'] || 50
+
+    progress_bar = ProgressBar.create(:total => GenericWork.count, format: "%t: |%B| %p%% %e")
+    solr_service_conn = ActiveFedora::SolrService.instance.conn
+    batch = []
+
+    GenericWork.find_each do |work|
+      batch << work.to_solr
+
+      if batch.count % add_batch_size == 0
```
Comment (hackmastera) [inline lib/tasks/chf_tasks.rake]: What happens to the last set of records if the total isn't a multiple of 50? Do we need an extra add outside of the loop to catch those?

Comment (jrochkind) [inline lib/tasks/chf_tasks.rake]: oops good catch!

Comment (jrochkind) [inline lib/tasks/chf_tasks.rake]: i love having a peer reviewer! didnt have that at jh really

Comment (hackmastera) [inline lib/tasks/chf_tasks.rake]: Yeah having people review your code is great!

**Pull request merged**